### ISSUE TITLE
Pdfviewer Bookmarks not translating correctly 

### ISSUE THREAD

**Issue by ShehabSN** [syncfusion/flutter-widgets#1143]
I have a pdf that has bookmarks in Arabic. I have used the localization package as can be seen in the code and that is working for the page number navigator but the bookmarks are not being translated correctly at all.

```
  runApp(const MaterialApp(
    title: 'Syncfusion PDF Viewer Demo',
    home: HomePage(),
    localizationsDelegates: [
      GlobalMaterialLocalizations.delegate,
      GlobalWidgetsLocalizations.delegate,
      GlobalCupertinoLocalizations.delegate,
      SfGlobalLocalizations.delegate
    ],
    supportedLocales: [
      Locale('ar'),
    ],
    locale: Locale('ar'),
  ));
```

How it looks in PDF
![correctbookmarks](https://user-images.githubusercontent.com/43920822/227359585-4a0959c1-c8a7-4363-83ef-479d52b9f485.PNG)

How it looks in app
![incorrectbookmarks](https://user-images.githubusercontent.com/43920822/227359651-2e45a9a6-2356-46e5-a0ba-30e02998ebae.jpeg)

None of the sub sections/bookmarks are translated correctly either. I've attached the pdf for reference. Any ideas how to resolve this or would I need to build a custom bookmark viewer? Thanks!

[Royce.pdf](https://github.com/syncfusion/flutter-widgets/files/11056223/Royce.pdf)

**Comment by deepika134** [comment]
Hi @ShehabSN , We have confirmed the issue "RTL text in a bookmark is not retrieved properly in a specific PDF document" as a defect, and the fix will be included in our upcoming weekly release, which will be available on April 4, 2023.

Use the below feedback link to track the status of the reported bug.
https://www.syncfusion.com/feedback/42353/rtl-text-in-bookmark-is-not-retrieved-properly-in-specific-pdf-document 

Disclaimer: “Inclusion of this solution in the weekly release may change due to other factors including but not limited to QA checks and works reprioritization.”

**Comment by deepika134** [comment]
Hi @ShehabSN , The issue " RTL text in a bookmark is not retrieved properly in a specific PDF document" has been resolved and the fix is available in the latest weekly (v21.1.38) release. Please find the link below.
https://pub.dev/packages/syncfusion_flutter_pdfviewer/versions/21.1.38
